tests/e2e/e2e_test.go

```diff
@@ -97,10 +96,21 @@ func init() {
 	)
 }
 
-var cli client.Client
+var (
+	cli client.Client
+	log logging.Logger
+)
 
 var _ = ginkgo.BeforeSuite(func() {
-	var err error
+	lvl, err := logging.ToLevel(logLevel)
+	gomega.Ω(err).Should(gomega.BeNil())
+	lcfg := logging.Config{
+		DisplayLevel: lvl,
+	}
+	logFactory := logging.NewFactory(lcfg)
+	log, err = logFactory.Make("test")
```

Comment:
is this controlling log path? I think not specified log paths should go to tmp, but also there should be some way to specify log path

Reply:
No this is just the log name.

Reply:
I added a configurable log path for e2e tests